### PR TITLE
multirocks related config update

### PR DESCRIPTION
**Pull request by tonyxuqqi**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v7.2 (TiDB 7.2 versions)
- [x] v7.1 (TiDB 7.1 versions)
- [ ] v7.0 (TiDB 7.0 versions)
- [ ] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: tikv-configuration-file.md [hunk 1]
```diff
@@ -2233,3 +2233,20 @@ Raft Engine 相关的配置项。
...
++ 控制某个region成为热点region的读qps的阈值
++ 默认值: 当`region-split-size`小于4GB时 `3000`; 否则是`7000`
+
+### `byte-threshold`
++ 控制某个region成为热点region的读吞吐的阈值
++ 默认值: 当`region-split-size`小于4GB时 `30MB`； 否则`100MB`
+
+### `region-cpu-overload-threshold-ratio`
++ 控制某个region成为热点region的CPU占用的阈值
++ 默认值: 当`region-split-size`小于4GB时 `0.25`； 否则`0.75`
```
Comment (hfxsd) [inline tikv-configuration-file.md:2252]:
```suggestion
Load Base Split 相关的配置项。

### `qps-threshold`

+ 控制某个 Region 成为热点 Region 的读 QPS 阈值。
+ 默认值：当 `region-split-size` 小于 4 GB 时，默认值为每秒 `3000` QPS; 大于或等于 4 GB 时，默认值为每秒 `7000` QPS。

### `byte-threshold`

+ 控制某个 Region 成为热点 Region 的读吞吐阈值。
+ 默认值：当 `region-split-size` 小于 4 GB 时，默认值为每秒 `30MiB` 流量；大于或等于 4 GB 时，默认值为每秒 `100MiB` 流量。

### `region-cpu-overload-threshold-ratio`

+ 控制某个 Region 成为热点 Region 的 CPU 占用阈值。
+ 默认值：当 `region-split-size` 小于 4 GB 时，默认值为 `0.25`；大于或等于 4 GB 时，默认值为 `0.75`。
```

File: configure-load-base-split.md [hunk 2]
```diff
@@ -31,9 +31,9 @@ Load Base Split 后的 Region 不会被迅速 Merge。一方面，PD 的 `MergeC
 
 目前的 Load Base Split 的控制参数如下：
 
-- `split.qps-threshold`：表明一个 Region 被识别为热点的 QPS 阈值，默认为每秒 `3000` QPS。
-- `split.byte-threshold`：自 v5.0 引入，表明一个 Region 被识别为热点的流量阈值，单位为 Byte，默认为每秒 `30 MiB` 流量。
-- `split.region-cpu-overload-threshold-ratio`：自 v6.2.0 引入，表明一个 Region 被识别为热点的 CPU 使用率（占读线程池 CPU 时间的百分比）阈值，默认为 `0.25`。
+- `split.qps-threshold`：表明一个 Region 被识别为热点的 QPS 阈值，当`region-split-size` 小于4GB时，默认为每秒 `3000` QPS；否则为每秒`7000`。
+- `split.byte-threshold`：自 v5.0 引入，表明一个 Region 被识别为热点的流量阈值，单位为 Byte，当`region-split-size` 小于4GB时,默认为每秒 `30 MiB` 流量;否则为每秒`100 MiB`。
+- `split.region-cpu-overload-threshold-ratio`：自 v6.2.0 引入，表明一个 Region 被识别为热点的 CPU 使用率（占读线程池 CPU 时间的百分比）阈值，当`region-split-size` 小于4GB时,默认为 `0.25`;否则为`0.75`。
```
Comment (hfxsd) [inline configure-load-base-split.md:36]:
```suggestion
- `split.qps-threshold`：表明一个 Region 被识别为热点的 QPS 阈值。当 `region-split-size` 小于 4 GB 时，默认为每秒 `3000` QPS；大于或等于 4 GB 时，默认值为每秒 `7000` QPS。
- `split.byte-threshold`：自 v5.0 引入，表明一个 Region 被识别为热点的流量阈值，单位为 Byte，当 `region-split-size` 小于 4GB 时，默认值为每秒 `30 MiB` 流量；大于或等于 4 GB 时，默认值为每秒 `100 MiB`。
- `split.region-cpu-overload-threshold-ratio`：自 v6.2.0 引入，表明一个 Region 被识别为热点的 CPU 使用率（占读线程池 CPU 时间的百分比）阈值，当 `region-split-size` 小于 4 GB 时，默认为 `0.25`；大于或等于 4 GB 时，默认值为 `0.75`。
```

**Review by tabokie — APPROVED** [review]

**Review by Oreoxmt — APPROVED** [review]

File: releases/release-7.1.0.md [hunk 3]
```diff
@@ -252,6 +252,9 @@ TiDB 版本：7.1.0 (upcoming)
 | -------- | -------- | -------- | -------- |
 | TiDB | [`force-init-stats`](/tidb-configuration-file.md#force-init-stats-从-v710-版本开始引入) | 新增 | 用于控制 TiDB 启动时是否在统计信息初始化完成后再对外提供服务。 |
 | TiDB | [`lite-init-stats`](/tidb-configuration-file.md#lite-init-stats-从-v710-版本开始引入) | 新增 | 用于控制 TiDB 启动时是否采用轻量级的统计信息初始化。 |
+| TiKV | [`split.byte-threshold`](/tikv-configuration-file.md#byte-threshold-从-v50-版本开始引入) | 修改 | 当 [`region-split-size`](/tikv-configuration-file.md#region-split-size) 大于等于 4GB 时，默认值从 `30MiB` 修改为 `100MiB`。 |
+| TiKV | [`split.qps-threshold`](/tikv-configuration-file.md#qps-threshold) | 修改 | 当 [`region-split-size`](/tikv-configuration-file.md#region-split-size) 大于等于 4GB 时，默认值从 `3000` 修改为 `7000`。 |
+| TiKV | [`split.region-cpu-overload-threshold-ratio`](/tikv-configuration-file.md#region-cpu-overload-threshold-ratio-从-v620-版本开始引入) | 修改 | 当 [`region-split-size`](/tikv-configuration-file.md#region-split-size) 大于等于 4GB 时，默认值从 `0.25` 修改为 `0.75`。 |
```
Comment (hfxsd) [inline releases/release-7.1.0.md:257]:
```suggestion
| TiKV | [`split.byte-threshold`](/tikv-configuration-file.md#byte-threshold-从-v50-版本开始引入) | 修改 | 当 [`region-split-size`](/tikv-configuration-file.md#region-split-size) 大于等于 4 GB 时，默认值从 `30MiB` 修改为 `100MiB`。 |
| TiKV | [`split.qps-threshold`](/tikv-configuration-file.md#qps-threshold) | 修改 | 当 [`region-split-size`](/tikv-configuration-file.md#region-split-size) 大于等于 4 GB 时，默认值从 `3000` 修改为 `7000`。 |
| TiKV | [`split.region-cpu-overload-threshold-ratio`](/tikv-configuration-file.md#region-cpu-overload-threshold-ratio-从-v620-版本开始引入) | 修改 | 当 [`region-split-size`](/tikv-configuration-file.md#region-split-size) 大于等于 4 GB 时，默认值从 `0.25` 修改为 `0.75`。 |
```

File: tikv-configuration-file.md [hunk 4]
```diff
@@ -2233,3 +2233,29 @@ Raft Engine 相关的配置项。
...
+
+## split
+
+[Load Base Split](/configure-load-base-split.md) 相关的配置项。
+
+### `byte-threshold` <span class="version-mark">从 v5.0 版本开始引入</span>
+
++ 控制某个 Region 被识别为热点 Region 的流量阈值。
++ 默认值：
+    + 当 [`region-split-size`](#region-split-size) 小于 4 GB 时，默认值为每秒 `30MiB` 流量。
```
Comment (hfxsd) [inline tikv-configuration-file.md:2245]:
```suggestion

    + 当 [`region-split-size`](#region-split-size) 小于 4 GB 时，默认值为每秒 `30MiB` 流量。
```

File: tikv-configuration-file.md [hunk 5]
```diff
@@ -2233,3 +2233,29 @@ Raft Engine 相关的配置项。
...
++ 默认值：
+
+    + 当 [`region-split-size`](#region-split-size) 小于 4 GB 时，默认值为每秒 `3000` QPS。
+    + 当 [`region-split-size`](#region-split-size) 大于或等于 4 GB 时，默认值为每秒 `7000` QPS。
+
+### `region-cpu-overload-threshold-ratio` <span class="version-mark">从 v6.2.0 版本开始引入</span>
+
++ 控制某个 Region 被识别为热点 Region 的 CPU 使用率阈值。
++ 默认值：
+    + 当 [`region-split-size`](#region-split-size) 小于 4 GB 时，默认值为 `0.25`。
```
Comment (hfxsd) [inline tikv-configuration-file.md:2260]:
```suggestion

    + 当 [`region-split-size`](#region-split-size) 小于 4 GB 时，默认值为 `0.25`。
```

**Review by hfxsd — APPROVED** [review]
Rest LGTM

File: configure-load-base-split.md [hunk 6]
```diff
@@ -31,9 +31,9 @@ Load Base Split 后的 Region 不会被迅速 Merge。一方面，PD 的 `MergeC
 
 目前的 Load Base Split 的控制参数如下：
 
-- `split.qps-threshold`：表明一个 Region 被识别为热点的 QPS 阈值，默认为每秒 `3000` QPS。
-- `split.byte-threshold`：自 v5.0 引入，表明一个 Region 被识别为热点的流量阈值，单位为 Byte，默认为每秒 `30 MiB` 流量。
-- `split.region-cpu-overload-threshold-ratio`：自 v6.2.0 引入，表明一个 Region 被识别为热点的 CPU 使用率（占读线程池 CPU 时间的百分比）阈值，默认为 `0.25`。
+- [`split.qps-threshold`](/tikv-configuration-file.md#qps-threshold)：表明一个 Region 被识别为热点的 QPS 阈值。当 [`region-split-size`](/tikv-configuration-file.md#region-split-size) 小于 4 GB 时，默认为每秒 `3000` QPS。当 `region-split-size` 大于或等于 4 GB 时，默认值为每秒 `7000` QPS。
+- [`split.byte-threshold`](tikv-configuration-file.md#byte-threshold-从-v50-版本开始引入)：自 v5.0 引入，表明一个 Region 被识别为热点的流量阈值，单位为 Byte。当 `region-split-size` 小于 4GB 时，默认值为每秒 `30 MiB` 流量。当 `region-split-size` 大于或等于 4 GB 时，默认值为每秒 `100 MiB` 流量。
```
Comment (Oreoxmt) [inline configure-load-base-split.md:35]:
```suggestion
- [`split.byte-threshold`](tikv-configuration-file.md#byte-threshold-从-v50-版本开始引入)：自 v5.0 引入，表明一个 Region 被识别为热点的流量阈值，单位为 Byte。当 `region-split-size` 小于 4 GB 时，默认值为每秒 `30 MiB` 流量。当 `region-split-size` 大于或等于 4 GB 时，默认值为每秒 `100 MiB` 流量。
```

File: configure-load-base-split.md [hunk 7]
```diff
@@ -31,9 +31,9 @@ Load Base Split 后的 Region 不会被迅速 Merge。一方面，PD 的 `MergeC
 
 目前的 Load Base Split 的控制参数如下：
 
-- `split.qps-threshold`：表明一个 Region 被识别为热点的 QPS 阈值，默认为每秒 `3000` QPS。
-- `split.byte-threshold`：自 v5.0 引入，表明一个 Region 被识别为热点的流量阈值，单位为 Byte，默认为每秒 `30 MiB` 流量。
-- `split.region-cpu-overload-threshold-ratio`：自 v6.2.0 引入，表明一个 Region 被识别为热点的 CPU 使用率（占读线程池 CPU 时间的百分比）阈值，默认为 `0.25`。
+- [`split.qps-threshold`](/tikv-configuration-file.md#qps-threshold)：表明一个 Region 被识别为热点的 QPS 阈值。当 [`region-split-size`](/tikv-configuration-file.md#region-split-size) 小于 4 GB 时，默认为每秒 `3000` QPS。当 `region-split-size` 大于或等于 4 GB 时，默认值为每秒 `7000` QPS。
+- [`split.byte-threshold`](tikv-configuration-file.md#byte-threshold-从-v50-版本开始引入)：自 v5.0 引入，表明一个 Region 被识别为热点的流量阈值，单位为 Byte。当 `region-split-size` 小于 4 GB 时，默认值为每秒 `30 MiB` 流量。当 `region-split-size` 大于或等于 4 GB 时，默认值为每秒 `100 MiB` 流量。
```
Comment (Oreoxmt) [inline configure-load-base-split.md:35]:
```suggestion
- [`split.byte-threshold`](/tikv-configuration-file.md#byte-threshold-从-v50-版本开始引入)：自 v5.0 引入，表明一个 Region 被识别为热点的流量阈值，单位为 Byte。当 `region-split-size` 小于 4 GB 时，默认值为每秒 `30 MiB` 流量。当 `region-split-size` 大于或等于 4 GB 时，默认值为每秒 `100 MiB` 流量。
```

**Pull request merged**